src/utils/equipment-types.js

```diff
@@ -87,6 +75,19 @@ const FreeProps = {
         type: filteredTypes.freeProperties,
     },
 };
+const FreePropsS = {
```

Comment:
S for substation ? you can be more explicit in naming

Reply:
We are happy with '1' and '2'. Why could we not be happy with just this 'S' that has just one local usage ?

Reply:
hum.. not so local, you have a corresponding component named FreePropertiesS